Modules/Core/Support/RedirectHelper.php

```
@@ -0,0 +1,49 @@
+ */
+class RedirectHelper
+{
+    /**
+     * Redirect the user to a given URL.
+     *
+     * @param string $fallback_url_string
+     * @param bool   $redirect
+     *
+     * @return mixed
+     */
+    public static function redirect_to($fallback_url_string, $redirect = true)
```

PHPMD (2.15.0)

The method redirect_to has a boolean flag argument $redirect, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)